gw_spaceheat/actors/actor_base.py

```diff
@@ -51,9 +52,10 @@ def __init__(self, node: ShNode, logging_on=False):
         self.consume_client.on_connect = self.on_consume_connect
         self.consume_client.on_connect_fail = self.on_consume_connect_fail
         self.consume_client.on_disconnect = self.on_consume_disconnect
-        self.consume_client.connect(self.mqttBroker)
         if self.logging_on:
             self.consume_client.on_log = self.on_log
+            self.consume_client.enable_logger()
+        self.consume_client.connect(self.mqttBroker)
```

Comment:
plus this

Reply:
Sorry, I don't follow. What do you mean?